src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+If a Revolutionary *or crewmember* dies by the hand of another crewmember (being dealt the damage that put them in a critical state), they will deal **15** non-split LHP in a **8** tile line-of-sight radius around their body. While remaining dead they will also provide a passive **-0.1** LHP / second in a **4** tile line-of-sight radius. This works in the same way but in reverse if a Revolutionary performs the kill, healing LHP upon death when killed by a Revolutionary/Crewmember. The initial burst also has a **120** second cooldown, to avoid repeatedly reviving and killing the same person twice to trigger the effect. This means both sides are discouraged from indiscriminately killing, favoring limited killing and kidnapping. 
+
+If the death of a *Revolutionary* is at the hand of a *crewmember*, the initial burst of LHP damage is able to convert. This is not true for the death of a crewmember by the hand of a crewmember (otherwise large-scale death in other gamemodes could risk spawning Revolutionaries without any headrev). 
+
+Being converted/deconverted by a character dying displays a special conversion message, to indicate it was the reason for being converted/deconverted.
+
+There is an interesting dichotomy that can form here where Revolutionaries have an easier time to "heal" revolutionary fervor by dealing LHP damage to each other via chanting codewords. On the other hand, Security has more access to stun weaponry and as such have an easier time subduing Revolutionary groups non-lethally. 
+
+### Mindshields
+
+Mindshield implants are a way to add extra safety for vital station personnel. They can not prevent or heal conversion on its own, but provides a buffer that requires the Revolutionaries to put in targeted effort to successfully convert someone with a mindshield. The mindshield is not a hard counter to Revolutionaries and while serving some use in preventing conversion they are not useful for deconversion.
+
+Command and Security are the only roles that have mindshields roundstart.
+
+- A mindshield gives its owner a "shield" with **100** LHP. The mindshield absorbs any LHP damage dealt to the owner and takes it instead.
+- The shield slowly regains LHP on its own, at **0.2** LHP / second. This functionality does not work when converted to the Revolution. 
+- Having a mindshield does not change Revolutionary status or antag faction; a Revolutionary can be mindshielded and still belong to the Revolution.
+- A newly injected mindshield always has full LHP, regardless of Revolutionary status.
+- Mindshields can be removed, much like any other implant. The exceptions for this is the Captain and ERT/CentComm, who have unremovable mindshields (however due to being unconvertable, this is mostly just flavor). 
+
+New mindshields are able to be purchased from Cargo at a reasonable price. One possible suggestion would be 4 implanters for $8000.
```

Comment:
I think giving perfect mindshielding to the Captain, HOS, Warden, Dets, Secoffs and Cadets and then just not mindshielding the other heads (or giving them these weaker ones) makes more sense to avoid shithousery. Arguably HOP too given how powerful AA is.

Reply:
Overall I'm really not a fan of keeping mindshields around at all anyway. It's an unsatisfying and slow mechanic and still requires "rev discovery" admin metashield rules that the admins hate enforcing. Mindshielding as an implant is just kinda bad design.

Feels more like something that could be done via surgery tho.

Reply:
I disagree with this paragraph. Yes; Sec still benefit from mindshielding crew, but it is no longer a binary guarantee. Additionally, a Rev could easily get a mindshield themselves and become a trusted ally, or the revs successfully convett someone with a shield - which would also be why Sec would no longer be able to rely on the hud icon exclusively.